var/spack/repos/builtin/packages/r-rmpi/package.py

```diff
@@ -38,18 +38,25 @@ class RRmpi(RPackage):
     depends_on('mpi')
     depends_on('r@2.15.1:')
 
-    def install(self, spec, prefix):
-        if 'mpich' in spec:
-            Rmpi_type = 'MPICH'
-        elif 'mvapich' in spec:
-            Rmpi_type = 'MVAPICH'
```

Comment:
@tmerrick1 I changed this section of the code a bit. `MVAPICH` isn't a supported keyword for `--with-Rmpi-type`. Also, I'm pretty sure we want `MPICH2`, not `MPICH`.